### PR TITLE
Add frame range options to gapit stats

### PR DESCRIPTION
**Pull request by iburinoc**
Options emulate those of gapit video:
  - frame-start is the frame to start at
  - frame-count is the number of frames to consider after the start

### CODE REVIEW

File: cmd/gapit/stats.go [hunk 1]
```diff
@@ -62,20 +64,62 @@ func loadCapture(ctx context.Context, flags flag.FlagSet, gapisFlags GapisFlags)
...
+	begin, end := len(events), len(events)
+	for i, e := range events {
+		if i < begin && e.Command.Indices[0] >= startIndex {
+			begin = i
+		}
+		if i < end && e.Command.Indices[0] >= endIndex {
+			end = i
+		}
+	}
+	fmt.Println(startIndex, endIndex, begin, end)
```
Comment (ben-clayton) [inline cmd/gapit/stats.go]: Is this debug?

Comment (iburinoc) [reply cmd/gapit/stats.go]: Yep, oops, will fix.

File: cmd/gapit/stats.go [hunk 2]
```diff
@@ -62,20 +64,61 @@ func loadCapture(ctx context.Context, flags flag.FlagSet, gapisFlags GapisFlags)
...
+
+	startIndex := fifIndices[verb.Frames.Start]
+	endIndex := uint64(math.MaxUint64)
+	if verb.Frames.Count >= 0 &&
+		verb.Frames.Start+verb.Frames.Count < len(fifIndices) {
+
+		endIndex = fifIndices[verb.Frames.Start+verb.Frames.Count]
+	}
+
+	begin, end := len(events), len(events)
```
Comment (ben-clayton) [inline cmd/gapit/stats.go]: I think this is equivalent to:

```go

begin := sort.Search(len(events), func(i int) bool {
	return events[i].Command.Indices[0] >= startIndex
})
end := sort.Search(len(events), func(i int) bool {
	return events[i].Command.Indices[0] >= endIndex
})
```

File: cmd/gapit/stats.go [hunk 3]
```diff
@@ -62,20 +64,61 @@ func loadCapture(ctx context.Context, flags flag.FlagSet, gapisFlags GapisFlags)
...
+}
+
+func (verb *infoVerb) Run(ctx context.Context, flags flag.FlagSet) error {
+	client, capture, err := loadCapture(ctx, flags, verb.Gapis)
+	if err != nil {
+		return err
+	}
+	defer client.Close()
+
+	events, err := verb.GetEventsInRange(ctx, client, capture)
```
Comment (ben-clayton) [inline cmd/gapit/stats.go]: Shouldn't this be `getEventsInRange` ?

File: cmd/gapit/stats.go (hunk 2 above)
Comment (iburinoc) [reply cmd/gapit/stats.go]: Changed, thanks

File: cmd/gapit/stats.go (hunk 3 above)
Comment (iburinoc) [reply cmd/gapit/stats.go]: Oops, fixed, thanks

**Pull request merged**